### PR TITLE
Migrate TC test_rebalance_two_volumes.py

### PR DESCRIPTION
**Pull request by Sheetalpamecha**
Migrate TC[ test_rebalance_two_volumes.py](https://github.com/gluster/glusto-tests/blob/master/tests/functional/dht/test_rebalance_two_volumes.py)

Updates: #292
Signed-off-by: Sheetal Pamecha <spamecha@redhat.com>

### CODE REVIEW

**Comment by Sheetalpamecha** [comment]
```
Results for dht
Test nature : disruptive
TestName : test_rebalance_two_volumes
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     dist    |     PASS    |        00:04:27       |      NA     |
|   dist-rep  |     PASS    |        00:07:24       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   1   |
|         Non Disruptive Tests         |   0   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   2   |
|            Tests Skipped             |   0   |
|             Tests Passed             |   2   |
|             Tests Failed             |   0   |
|             Total Tests              |   1   |
|   Disruptive Tests Pass Percentage   | 100.0 |
| Non Disruptive Tests Pass Percentage |   0   |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 13 minutes 57 seconds
✔ Tearing down successful.
```

File: tests/functional/dht/test_rebalance_two_volumes.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import traceback
+from copy import deepcopy
+from tests.d_parent_test import DParentTest
+
+
+class TestRebalanceTwoVolumes(DParentTest):
+
+    def terminate(self):
+        try:
+            self.redant.cleanup_volume(self.server_list, self.second_vol_name)
```
Comment (nik-redhat) [inline tests/functional/dht/test_rebalance_two_volumes.py:29]: We can avoid this, as the terminate function will take care of it by default for disruptive TCs.

File: tests/functional/dht/test_rebalance_two_volumes.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                                                      factor)
+        redant.volume_create_with_custom_bricks(self.second_vol_name,
+                                                self.server_list[0],
+                                                conf_dict, brick_cmd,
+                                                brick_dict, force=True)
+        redant.volume_start(self.second_vol_name, self.server_list[0])
+        self.redant.execute_abstract_op_node("mkdir -p "
+                                             f"{self.second_mountpoint}",
+                                             self.client_list[0])
+        self.redant.volume_mount(self.server_list[0], self.vol_name,
```
Comment (nik-redhat) [inline tests/functional/dht/test_rebalance_two_volumes.py:69]: It should be second volume as well. `self.second_volname`

**Review by nik-redhat — CHANGES_REQUESTED** [review]

File: tests/functional/dht/test_rebalance_two_volumes.py (hunk 2 above)
Comment (Sheetalpamecha) [reply tests/functional/dht/test_rebalance_two_volumes.py]: done

File: tests/functional/dht/test_rebalance_two_volumes.py (hunk 1 above)
Comment (Sheetalpamecha) [reply tests/functional/dht/test_rebalance_two_volumes.py]: okay

**Comment by Sheetalpamecha** [comment]
```
Results for dht
Test nature : disruptive
TestName : test_rebalance_two_volumes
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     dist    |     PASS    |        00:13:03       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   1   |
|         Non Disruptive Tests         |   0   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   1   |
|            Tests Skipped             |   0   |
|             Tests Passed             |   1   |
|             Tests Failed             |   0   |
|             Total Tests              |   1   |
|   Disruptive Tests Pass Percentage   | 100.0 |
| Non Disruptive Tests Pass Percentage |   0   |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 23 minutes 53 seconds
✔ Tearing down successful.
```

**Review by nik-redhat — APPROVED** [review]

**Pull request merged**